docs/api-docs/tax/tax-settings.md

```diff
@@ -0,0 +1,92 @@
+# Tax Settings
+
+Tax settings allow you to choose how a store handles tax calculations and price displays. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```

Comment:
> Tax settings allow you to choose how a store handles tax calculations and price displays.

I'd go the reverse angle and remove the pluralisation here.

Suggestion:

> Tax settings allow you to choose how a store handles tax calculation and price display.

It is worth noting that we plan to expand this endpoint to do more things in the future, beyond just these two. I suppose we can add a catch all when that day comes?